README.md

```diff
@@ -29,6 +29,152 @@ $ yarn install
 $ yarn e2e
  ```
 
+## Running the tests locally using docker
+
+Tests can be executed against an instance of CCD running locally using docker.
+
+Follow the instructions here to get CCD running locally:
+
+https://github.com/hmcts/ccd-docker
+
+Once all of the docker containers are running, add the following user roles to the IDAM database:
+
+```
+psql -h 127.0.0.1 -p 5000 -U postgres -d idam -c \
```

Comment:
I think it would be useful to actually add these commands for setting up the docker containers as a script, so that changes can be tracked and we don't forget to update the ReadMe with any tweaks

Reply:
Yep, some of this could go into a parameterized script